.github/workflows/cabal.project.local.ci

```diff
@@ -0,0 +1,5 @@
+package cardano-crypto-praos
```

Comment:
Do we really need `cabal.project.local` with some unrelated 3rd-party package here?